src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -539,15 +539,16 @@ ov::AnyMap get_default_generate_config(const std::shared_ptr<ov::Model>& model,
     }
     // NB: Unconditionally set for generation model
     config.emplace("NPUW_DQ", "YES");
+    if (npudesc.has_value() && npudesc->compiler_dq) {
+        config.emplace("NPUW_DQ_FULL", "NO");
+        config.emplace("NPU_COMPILER_DYNAMIC_QUANTIZATION", true);
+    }
```

Comment:
You certainly don't need to make it twice, just do this in the shared section of the config (so it goes to both prefill and kvcache).

Reply:
Also, when you switch to compiler DQ, you'll have to disable DCOFF since otherwise DCOFF will be applied to this model and it will run in FP16.

Reply:
Done